packages/hdwallet-provider/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import * as bip39 from 'bip39';
+import * as EthUtil from 'ethereumjs-util';
+import ethJSWallet from 'ethereumjs-wallet';
+import EthereumHDKey from 'ethereumjs-wallet/hdkey';
+import debug from 'debug';
+import { Transaction } from 'ethereumjs-tx';
+import ProviderEngine from 'web3-provider-engine';
+import FiltersSubprovider from 'web3-provider-engine/subproviders/filters';
+import NonceSubProvider from 'web3-provider-engine/subproviders/nonce-tracker';
+import HookedSubprovider from 'web3-provider-engine/subproviders/hooked-wallet';
+import ProviderSubprovider from 'web3-provider-engine/subproviders/provider';
+import Url from 'url';
+import Web3 from 'web3';
+import {
+  JSONRPCRequestPayload,
+  JSONRPCResponsePayload
+} from 'ethereum-protocol';
+
+const debugInstance = debug('@truffle/hdwallet-provider');
```

Comment:
In other projects we usually do `import debugModule from "debug"` so we can preserve `debug` as the variable name that's used everywhere

Reply:
This is exactly what I wanted to come to, thanks for the help